test/test.vimspec

```diff
@@ -1093,4 +1095,79 @@ Describe selection=exclusive
     End
 End
 
+Describe g:clever_f_mark_direct
+    Before
+        new
+        let g:clever_f_mark_direct = 1
+        call clever_f#reset()
+        highlight link CleverFDirect CleverFDefaultLabel
+        call AddLine('pOge huga Hiyo poyo')
```

Comment:
Please add tests which confirm it works fine with Japanese text

Reply:
OK